.circleci/config.yml

```diff
@@ -112,7 +113,7 @@ jobs:
 
       - run:
           name: Verify screenshots and generate diffs and new screenshots
-          command: node dockers/screenshotter/screenshotter.js --selenium-ip localhost -b $CIRCLE_JOB --verify --diff --new --coverage
+          command: yarn node dockers/screenshotter/screenshotter.js --selenium-ip localhost -b $CIRCLE_JOB --verify --diff --new --coverage
```

Comment:
https://yarnpkg.com/getting-started/install:
> Using a single package manager across your system has always been a problem. To be stable, installs need to be run with the same package manager version across environments, otherwise there's a risk we introduce accidental breaking changes between versions - after all, that's why the concept of lockfile was introduced in the first place! And with Yarn being in a sense your very first project dependency, it should make sense to "lock it" as well.
>
> For this reason, Yarn 2 and later are meant to be managed on a by-project basis. Don't worry, little will change!

Reply:
Is `yarn` monkey patching `node`?

Reply:
@kevinbarabash Sorry, I've quoted the wrong documentation. https://yarnpkg.com/advanced/migration#call-your-scripts-through-yarn-node-rather-than-node:
> We now need to **inject some variables into the environment** for Node to be able to locate your dependencies. In order to make this possible, we ask you to use `yarn node` which transparently does the heavy lifting.
>
> Note: this section only applies to the _shell CLI_. The commands defined in your `scripts` are unaffected, as we make sure that `node` always points to the right location, with the right variables already set.

Reply:
Looking at the source, it seems it just adds `--require .pnp.js` to `NODE_OPTIONS` variable.